src/util/rlog/logger.rs

```diff
@@ -1,60 +1,71 @@
-use log::{debug, error, info};
+use std::fmt;
+use str_macro::str;
 
-// Logger is a low-tech context logger. To add context call add_context().
+// LogContext is a low-tech log context. To add context call add_context().
 // To pass it around clone() it. If you can't do that then pass its
 // context() as a String and call new_from_context() to reconstitute it.
 //
 // TODO support dynamically setting log level here.
 #[derive(Clone)]
-pub struct Logger {
+pub struct LogContext {
     context: String,
 }
 
+impl fmt::Display for LogContext {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.context)
+    }
+}
+
 // Short, convenience method for tests to instantiate a new logger when
 // they don't care about passing context around.
-pub fn log() -> Logger {
-    Logger::new()
+pub fn context() -> LogContext {
```

Comment:
i dont think this is used, should prolly delete given that it will certainly be confused with the context() method below.